examples/rust/Earthfile

```diff
@@ -1,29 +1,19 @@
-VERSION 0.7
-FROM rust:1.72
-WORKDIR /rustexample
-
-install-chef:
-    RUN cargo install --debug cargo-chef
+VERSION --global-cache 0.7
+IMPORT github.com/earthly/lib/rust AS rust
 
-prepare-cache:
-    FROM +install-chef
-    COPY --dir src Cargo.lock Cargo.toml .
-    RUN cargo chef prepare
-    SAVE ARTIFACT recipe.json
-
-# Using cutoff-optimization to ensure cache hit (see examples/cutoff-optimization)
-build-cache:
-    FROM +install-chef
-    COPY +prepare-cache/recipe.json ./
-    CACHE target
-    RUN cargo chef cook --release
+FROM rust:slim-buster
+WORKDIR /rustexample
 
+# Cache dependencies
```

Comment:
would be nice to use comments that work with our earthly doc command.
e.g. 
```suggestion
# build creates a docker image using cached rust commands
```

Reply:
oh yeah. Good idea